rasa/nlu/selectors/response_selector.py

```diff
@@ -72,6 +71,7 @@
     TENSORBOARD_LOG_LEVEL,
     CONCAT_DIMENSION,
     FEATURIZERS,
+    DENSE_DIMENSION,
```

Comment:
in response selector, `128` is not enough?

Reply:
since we changed it to predict labels, it is basically another DIET

Reply:
When I set it to something lower, the performance dropped (see the tables above).